This patch is for VirtualBox-4.2.2, to temporarily remove references to the
fa_IR language file from the source tree, as it somehow got lost during tarball
creation. For more information on this subject please have a look at:

https://www.virtualbox.org/pipermail/vbox-dev/2012-October/005408.html

diff --git a/src/VBox/Frontends/VirtualBox/VBoxUI.pro b/src/VBox/Frontends/VirtualBox/VBoxUI.pro
index ffeb410..37ac56a 100644
--- a/src/VBox/Frontends/VirtualBox/VBoxUI.pro
+++ b/src/VBox/Frontends/VirtualBox/VBoxUI.pro
@@ -63,7 +63,6 @@ TRANSLATIONS = \
 	nls/VirtualBox_es.ts \
 	nls/VirtualBox_eu.ts \
 	nls/VirtualBox_fi.ts \
-	nls/VirtualBox_fa_IR.ts \
 	nls/VirtualBox_fr.ts \
 	nls/VirtualBox_gl_ES.ts \
 	nls/VirtualBox_hu.ts \
diff --git a/src/VBox/Frontends/VirtualBox/nls/ApprovedLanguages.kmk b/src/VBox/Frontends/VirtualBox/nls/ApprovedLanguages.kmk
index 66f0ec3..2fc19b9 100644
--- a/src/VBox/Frontends/VirtualBox/nls/ApprovedLanguages.kmk
+++ b/src/VBox/Frontends/VirtualBox/nls/ApprovedLanguages.kmk
@@ -23,7 +23,6 @@ VBOX_APPROVED_GUI_LANGUAGES := \
 	de    \
 	en    \
 	es    \
-	fa_IR \
 	hu    \
 	it    \
 	ja    \
